doc/faq.rst

```diff
@@ -34,12 +34,15 @@ An ECC key is roughly 500 bytes formatted in Base64 and RSA 2048 key is
 we opted for readability.
 
 
-Why do you drop all headers if there is more than one?
-------------------------------------------------------
-
-Because of multi-agent usage we may have to handle an inconsistent stream of
-headers already. Making this an inconsistent stream of multiple keys with
-priorities sounds like a lot of pain.
+Why do you ask to drop all headers if there is more than one?
+-------------------------------------------------------------
+
+Because of multi-agent usage we may have to handle an inconsistent
+stream of headers already. Making this an inconsistent stream of
+multiple headers per mail sounds like a lot of pain. We could come up
```

Comment:
actually, i never could quite follow this reasoning or this wording.  How does "an inconsistent stream of multiple headers" come about in the "there is more than one header" case?  Shouldn't this answer rather be about the difficulty of choosing one of multiple headers?  Let's say i get two valid Autocrypt headers in one mail and i choose which one i am going to use randomly -- what exactly would be the problem?